components/pedigree/resources/src/main/resources/pedigree/view/personVisuals.js

```diff
@@ -1055,6 +1055,12 @@ define([
                 }
             }
 
+            // move white background up if it is not the first child element of <svg>, otherwise it will hide labels
```

Comment:
you're right, I placed it at the place where the nodes are actually shifted. I better move it up in stack.